openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -64,7 +66,7 @@ def _prepare_ipi_cluster(self) -> None:
             self._create_install_config_file()
 
     def _ipi_download_installer(self) -> None:
-        openshift_install_str = "openshift-install"
+        openshift_install_str = f"openshift-install{'-fips' if self.cluster_info['fips'] else ''}"
```

Comment:
`openshift_install_str` represents `openshift-install` as str.
either include your code when we call the command (line 79) or save it on a new content and use it in the command.

Reply:
or rename to `openshift_install_binary`